Makefile

```diff
@@ -0,0 +1,39 @@
+GIT_TAG=$(shell git describe --exact-match HEAD)
+GIT_REPO=$(REPO_OWNER)/$(REPO_NAME)
+
+.PHONY: check
+check:
+	./scripts/checker.sh --helm-lint --directories ./bundles --helm-version v2.10.0
+
+.PHONY: convert
+convert:
+    ./scripts/convert.sh
+
+.PHONY: generate-changelog
+generate-changelog:
+    ifneq($(strip $(GIT_TAG)),)
+    ifneq($(strip $(GIT_TAG)),latest)
+    ./scripts/generate_changelog.sh
```

Comment:
generate_changelog uses travis env variables

Reply:
fixed for all scripts